### DISCUSSION TITLE
Gantt Chart

### DISCUSSION THREAD

**Discussion by rmotto75** [team-moeller/better-access-charts#140] [Q&A]
Thank you for the demo, the solution is fantastic and definitely an improvement compared to Modern Charts.

I was wondering if there is a way to adapt the chart.js library to display a Gantt chart. 

I would like to represent tasks and their duration on the chart (providing start date and end date or datediff in days). 

Thank you.

**Comment by team-moeller** [comment]
Hello!
When I first read your request, I thought directly about some plugins. A valuable resource for plugins is this site: https://github.com/chartjs/awesome. But sadly, there is no plugin for Gantt charts.
After a short search I found this video: https://www.youtube.com/watch?v=InMnGwP7iX4
It looks like this should solve your problem.
Have fun!
